doc/nrf/known_issues.rst

```diff
@@ -1849,7 +1898,7 @@ tx_buffer_length set incorrectly
 Trusted Firmware-M (TF-M)
 *************************
 
```

Comment:
NCSDK-13530: TF-M Minimal build has increased in size.
TF-M Minimal build exceeds 32kB due to increased dependencies in the TF-M crypto partition.
Affects: 1.9

Reply:
Added.